stl/inc/deque

```diff
@@ -22,22 +22,14 @@ _STL_DISABLE_CLANG_WARNINGS
 #undef new
 
 _STD_BEGIN
-// DEQUE PARAMETERS
-#define _DEQUEMAPSIZ 8 // minimum map size, at least 1
-#define _DEQUESIZ                               \
-    (sizeof(value_type) <= 1                    \
-            ? 16                                \
-            : sizeof(value_type) <= 2           \
-                  ? 8                           \
-                  : sizeof(value_type) <= 4 ? 4 \
-                                            : sizeof(value_type) <= 8 ? 2 : 1) // elements per block (a power of 2)
-
 // CLASS TEMPLATE _Deque_unchecked_const_iterator
 template <class _Mydeque>
 class _Deque_unchecked_const_iterator {
 private:
     using _Size_type = typename _Mydeque::size_type;
 
+    static constexpr int _DEQUESIZ = _Mydeque::_DEQUESIZ;

```

Comment:
I hate that these names are still in `_RESERVED_MACRO_IDENTIFIER` space. Would changing them to e.g. `_Deque_size` be in scope?

Reply:
I can do that as an isolated commit in this PR for ease of reviewing, yes.